js-exercises/object-utils/objectUtils.js

```diff
@@ -0,0 +1,28 @@
+function getKeyDescriptorPairs(object) {
+  return Object.getOwnPropertyNames(object).map((key) => {
+    const desc = Object.getOwnPropertyDescriptors(object, key);
+    return [key, desc];
+  });
+}
+
+function fromKeyDescPairs(pairs) {
+  return pairs.reduce((result, [key, desc]) => {
+    Object.defineProperties(result, key, desc);
+    return result;
+  }, {});
+}
+function map(object, cb) {
+  const pairs = getKeyDescriptorPairs(object);
```

Comment:
variable name pairs does not convey much information about the kind and thus can be named better